students_db/src/main/java/application/models/AnyObject.java

```diff
@@ -0,0 +1,14 @@
+package application.models;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@NoArgsConstructor
+@Getter
+@Setter
+public class AnyObject {
```

Comment:
i suggest rename it in AbstractEntity or something like this, *any object* is wider than entity from database